pydatastructs/trees/fenwick_tree.py

```diff
@@ -0,0 +1,34 @@
+__all__ = ['FenwickTree']
+class FenwickTree():
+    def __init__(self, array):
```

Comment:
We use `__new__` to create and initialise objects. See other modules and files for examples.